tests/transformers/test_transformers_model_export.py

```diff
@@ -1845,9 +1845,10 @@ def test_summarization_pipeline(summarizer_pipeline, model_path, data):
             "That gym smells like feet, hot garbage, and sadness",
             "I love that we have a moon",
         ],
+        [{"text": "test1", "text_pair": "test2"}],
+        [{"text": "test1", "text_pair": "pair1"}, {"text": "test2", "text_pair": "pair2"}],
     ],
 )
-@pytest.mark.skipif(RUNNING_IN_GITHUB_ACTIONS, reason=GITHUB_ACTIONS_SKIP_REASON)
```

Comment:
We need to run this in CI. We're already running this pipeline type elsewhere anyway.